src/Discord.Net.Core/Entities/Guilds/GuildFeature.cs

```diff
@@ -14,92 +14,172 @@ public enum GuildFeature
         /// </summary>
-        AnimatedIcon = 1 << 0,
+        AnimatedIcon = 1 << 1,
         /// <summary>
         ///     The guild has access to set a guild banner image.
         /// </summary>
-        Banner = 1 << 1,
+        Banner = 1 << 2,
+        /// <summary>
+        ///     The guild has access to channel banners.
+        /// </summary>
+        ChannelBanner = 1 << 3,
         /// <summary>
         ///     The guild has access to use commerce features (i.e. create store channels).
         /// </summary>
-        Commerce = 1 << 2,
+        Commerce = 1 << 4,
         /// <summary>
         ///     The guild can enable welcome screen, Membership Screening, stage channels and discovery, and receives community updates.
         /// </summary>
-        Community = 1 << 3,
+        Community = 1 << 5,
         /// <summary>
         ///     The guild is able to be discovered in the directory.
         /// </summary>
-        Discoverable = 1 << 4,
+        Discoverable = 1 << 6,
+        /// <summary>
+        ///     The guild has discoverable disabled.
+        /// </summary>
+        DiscoverableDisabled = 1 << 7,
+        /// <summary>
+        ///     The guild has enabled discoverable before.
+        /// </summary>
+        EnabledDiscoverableBefore = 1 << 8,
         /// <summary>
         ///     The guild is able to be featured in the directory.
         /// </summary>
-        Featureable = 1 << 5,
+        Featureable = 1 << 9,
+        /// <summary>
+        ///     The guild has a force relay.
+        /// </summary>
+        ForceRelay = 1 << 10,
+        /// <summary>
+        ///     The guild has a directory entry.
+        /// </summary>
+        HasDirectoryEntry = 1 << 11,
+        /// <summary>
+        ///     The guild is a hub.
+        /// </summary>
+        Hub = 1 << 12,
+        /// <summary>
+        ///     You shouldn't be here...
+        /// </summary>
+        InternalEmployeeOnly = 1 << 13,
         /// <summary>
         ///     The guild has access to set an invite splash background.
         /// </summary>
-        InviteSplash = 1 << 6,
+        InviteSplash = 1 << 14,
+        /// <summary>
+        ///     The guild is linked to a hub.
+        /// </summary>
+        LinkedToHub = 1 << 15,
+        /// <summary>
+        ///     The guild has member profiles.
+        /// </summary>
+        MemberProfiles = 1 << 16,
         /// <summary>
         ///     The guild has enabled <seealso href="https://discord.com/developers/docs/resources/guild#membership-screening-object">Membership Screening</seealso>.
         /// </summary>
-        MemberVerificationGateEnabled = 1 << 7,
+        MemberVerificationGateEnabled = 1 << 17,
         /// <summary>
         ///     The guild has enabled monetization.
         /// </summary>
-        MonetizationEnabled = 1 << 8,
+        MonetizationEnabled = 1 << 18,
+        /// <summary>
+        ///     The guild has more emojis
```

Comment:
```suggestion
        ///     The guild has more emojis.
```

Just for consistency